lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -4168,18 +4172,29 @@ class AdjointEmitter final : public SILInstructionVisitor<AdjointEmitter> {
       }
       domOrder.pushChildren(bb);
     }
-
+    
     // Create adjoint blocks and arguments, visiting original blocks in
     // post-order.
     for (auto *origBB : postOrderInfo->getPostOrder()) {
       auto *adjointBB = adjoint.createBasicBlock();
       adjointBBMap.insert({origBB, adjointBB});
-      // If adjoint block is the adjoint entry, continue.
-      if (adjointBB->isEntry())
-        continue;
-      // Otherwise, add a pullback struct argument to the adjoint block.
       auto pbStructLoweredType =
           remapType(getPullbackInfo().getPullbackStructLoweredType(origBB));
+      // If the BB is the original exit, then the adjoint block that we just
```

Comment:
There's a typo: `createed`.
I'd suggest a more succinct comment: `For the adjoint entry, create entry arguments and continue.`

Reply:
Thanks. Done.